docroot/sites/all/themes/custom/wsdot_classic/css/styles.css

```diff
@@ -105,6 +105,7 @@ input.searchButton {height: 23px;margin: 0;font-family: "trebuchet ms",helvetica
 .mainnav {width:955px;margin:0 auto;}
 .mainnav ul {list-style:none;background: transparent url(../images/back/tabs_div.png) no-repeat 0 8px;margin:0 auto;padding:5px 0 0 2px;width:955px;height:48px;}
 .mainnav ul li {float:left;font-weight:bold;margin-top:2px;height:40px;position:relative;}
+.mainnav ul li ul li {font-size: 1em;}
```

Comment:
This affects the top navigation not the main body content. Styles for that are under **#main**